docs/worldgen/biomemodifier.md

```diff
@@ -0,0 +1,741 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Biome Modifiers
+
+Biome Modifiers are a data-driven system that allows for changing many aspects of a biome. Ranging from injecting or removing PlacedFeatures, adding or deleting mob spawns, changing the climate, and adjusting foliage and water color. NeoForge provides several default Biome Modifiers that covers the majority of use cases for both players and modders.
+
+### Recommended Section To Read:
+
+- Players or Pack Makers:
+  - '[Applying Biome Modifiers](#Applying-Biome-Modifiers)'
+  - '[Built-in Neoforge Biome Modifiers](#Builtin-Neoforge-Biome-Modifiers)'
+
+
+- Modders doing simple additions or removal biome modifications:
+  - '[Applying Biome Modifiers](#Applying-Biome-Modifiers)'
+  - '[Built-in Neoforge Biome Modifiers](#Builtin-Neoforge-Biome-Modifiers)'
+  - '[Datagenning Biome Modifiers](#Datagenning-Biome-Modifiers)'
+
+
+- Modders who want to do custom or complex biome modifications:
+  - '[Applying Biome Modifiers](#Applying-Biome-Modifiers)'
+  - '[Creating Custom Biome Modifiers](#Creating-Custom-Biome-Modifiers)'
+  - '[Datagenning Biome Modifiers](#Datagenning-Biome-Modifiers)'
+
+
+## Applying Biome Modifiers:
+
+Biome Modifiers are like a set of modifications to apply to a biome when the game loads. To have NeoForge load a Biome Modifier JSON file into the game, the file will need to be under `data/<modid>/neoforge/biome_modifier/<path>.json` folder in the mod's resources or in a Datapack. Then once NeoForge loads the Biome Modifier, it'll read its instructions and apply the described modifications to all target biomes when the world is loaded up. Pre-existing Biome Modifiers from mods can be overridden by Datapacks having a new JSON file at the exact same location and name.
```

Comment:
The first sentence is redundant, so it can be removed. What I wanted is more or less explained better in the subsequent paragraph.